### ISSUE TITLE
Customize default systems for all scenes

### ISSUE THREAD

**Issue by mattjennings** [excaliburjs/Excalibur#3101]
### Context

It would be helpful to configure the default systems to be used for all scenes. This spares having to make a base scene class with the system added if I want to make sure it's always used.

### Proposal

```ts
// proposal 1
new ex.Engine({
   world: {
       systems: [...], // adds to current systems
       removeDefaultSystems: true/false // removes default if true
    }
})

// proposal 2
new ex.Engine({
   world: {
       systems: [...], // overrides systems, must add back ex.MotionSystem etc
    }
})
```

**Comment by eonarheim** [comment]
I'm into this, I'm leaning towards #1 because re-adding systems seems really rough... but on the other hand that's only once?

**Comment by mattjennings** [comment]
I agree, i feel like you'd need to expose the default systems somewhere to they can be spread in. Here's 2 more ideas:

```ts
// proposal 3
new ex.Engine({
   world: {
       systems: [...ex.DefaultSystems, MySystem],
    }
})

// proposal 4
new ex.Engine({
   world:  {
       setup(world) {
           world.systemManager.addSystem(MySystem)
       }
   }
})
```

**Comment by Autsider666** [comment]
I like the `removeDefaultSystems` option from proposal 1 a lot, because it currently takes a bit of work to get a scene without the default systems preloaded.

Using a callback (proposal 4) feels like the "best" method to me, because it prevents the reuse of system instances between scenes (causing `initialize` to be called more than once).

What about adding `initializeWorld: (world:World)=>void` to the scene constructor that falls back to a default function that adds the systems [currently added in the constructor](https://github.com/excaliburjs/Excalibur/blob/main/src/engine/Scene.ts#L166).
This default function could be overwritten if needed using something like proposal 4
